content/blog/2022/czi-global-communities-proposal/index.md

```diff
@@ -47,14 +47,14 @@ For this reason, our goal in this grant is to **provide human and technical serv
 It defines four major areas of collaboration:
 
 - **Cloud infrastructure management** - to facilitate access to cloud resources via managed cloud services that integrate open source tools.
-- **Application guidance and training** - to provide community leaders with the skills to utilize this infrastructure for the needs. This includes language-localized content and training materials.
+- **Application guidance and training** - to provide community leaders with the skills to utilize this infrastructure for their needs. This includes language-localized content and training materials.
 - **Training for trainers** - to provide community leaders with skills to share these workflows with others in their communities.
 - **Community leadership and management** - to provide community leaders with skills to sustain and grow healthy communities of practice.
 
 Over the course of two years, the team will provide a combination of the services described above for communities in Latin America and Africa, with the goal of understanding how such services can be most useful for these communities, how we can structure them to provide community representation in the direction of these services, and how we can sustain and scale this model of community-focused services for a global community.
```

Comment:
Provided this proposal is funded, over the course of two years, the team will provide a combination of the services described above for communities in Latin America and Africa, with the goal of understanding how such services can be most useful for these communities, how we can structure them to provide community representation in the direction of these services, and how we can sustain and scale this model of community-focused services for a global community.

Reply:
Or something along these lines reminding the reader that this is not yet funded :-)

Reply:
good call